sql-cli/sql_cli/templates/gen_tasks_dag.py.jinja2

```diff
@@ -14,6 +15,10 @@ with DAG(
     start_date=timezone.parse("{{ dag.start_date }}"),
     schedule_interval=None,
 ) as dag:
-    {%- for sql_file in dag.sorted_sql_files() %}
-    {{ define_task(sql_file, dag)|indent(width=4) }}
+    {%- for workflow_file in dag.sorted_workflow_files() %}
+    {% if workflow_file.__class__.__name__ == 'SqlFile'  %}
+    {{ define_transform_task(workflow_file, dag)|indent(width=4) }}
+    {% elif workflow_file.operator == 'load_file'  %}
+    {{ define_load_file_task(workflow_file, dag, workflow_file.get_yaml_content(), workflow_file.yaml_file_name)|indent(width=4) }}
```

Comment:
```suggestion
    {% elif workflow_file.__class__.__name__ == 'YamlFile'  %}
    {{ define_yaml_task(workflow_file, dag)|indent(width=4) }}
```
Can we make this work^?

Because then we would not need to add this for every operator.

Reply:
I think would be good to keep operator level macros to keep it easier to maintain.